### PR TITLE
docs(adr-init): initialized the `architecture-decisions` directory for ADRs

### PR DESCRIPTION
**Pull request by travi**
using the `adr init` command of `adr-tools` from https://github.com/npryce/adr-tools

### CODE REVIEW

**Comment by travi** [comment]
if we decide to merge this, i can follow up with some records to capture some of the discussions we've had recently. since these are numbered, it's probably worth doing them one at a time until we figure out if there are better ways.

i like the idea of using the `adr-tools` as a wizard for these, at least initially, but that seems to require the dotfile that informs the tool of the directory where the records are to be added

File: architecture-decisions/0001-record-architecture-decisions.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+Date: 2020-11-16
+
+## Status
+
+Accepted
+
+## Context
+
+We need to record the architectural decisions made on this project.
```
Comment (gr2m) [inline architecture-decisions/0001-record-architecture-decisions.md:11]: should we add that we discussed it between the two of us? Either here or in the `Decision` section?

Comment (travi) [reply architecture-decisions/0001-record-architecture-decisions.md:11]: yeah, i could add something. this was simply the result of what the tool generates by default, but a little more context would probably be worthwhile

File: architecture-decisions/0001-record-architecture-decisions.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+Date: 2020-11-16
+
+## Status
+
+Accepted
+
+## Context
+
+As we take steps to modernize CI and dependency updating, @gr2m and @travi have taken steps to discuss trade-offs of various options. During these discussions, the value of capturing the decisions from these discussions came up. Interest from both of us in ADRs was mentioned.
```
Comment (gr2m) [inline architecture-decisions/0001-record-architecture-decisions.md:11]:
```suggestion
As we take steps to modernize CI and dependency updating, [@gr2m](https://github.com/gr2m/) and [@travi](https://github.com/travi/) have taken steps to discuss trade-offs of various options. During these discussions, the value of capturing the decisions from these discussions came up. Interest from both of us in ADRs was mentioned.
```

**Review by gr2m — APPROVED** [review]
very cool, thanks for getting it started!

File: architecture-decisions/0001-record-architecture-decisions.md (hunk 1 above)
Comment (travi) [reply architecture-decisions/0001-record-architecture-decisions.md]: i added a bit more. happy to tweak further if you have thoughts on what else might be valuable to capture

File: architecture-decisions/0001-record-architecture-decisions.md (hunk 2 above)
Comment (travi) [reply architecture-decisions/0001-record-architecture-decisions.md]: i guess ive been spoiled by https://github.com/remarkjs/remark-github and forgot that doesnt work by default

**Pull request merged**